File: programme/area-of-trapezoid/area-of-trapezoid.c
/* Program to Find the area of trapezoid */

#include <stdio.h>

int main()
{
    float base1, base2, height;
    float area;

    printf("Base 1 : ");
    scanf("%f", &base1);

    printf("Base 2 : ");
    scanf("%f", &base2);

    printf("Height : ");
    scanf("%f", &height);

    area = (base1 + base2) * height / 2;

    printf("\nArea: %.2f\n", area);
    return 0;
}

File: programme/area-of-trapezoid/AreaOfTrapezoid.java
/* program to calculate area of trapezoid */

import java.util.Scanner;

public class AreaOfTrapezoid {

    public static double area(double base1, double base2, double height) {
        return ((base1 + base2) / 2) * height;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Base 1 : ");
        double base1 = sc.nextDouble();
        System.out.print("Base 2 : ");
        double base2 = sc.nextDouble();
        System.out.print("Height : ");
        double height = sc.nextDouble();
        System.out.println("\nArea : " + area(base1, base2, height));
        sc.close();
    }
}
